### PR TITLE
Accept apt

### PR DESCRIPTION
**Pull request by MikeBLambert**
Nannies can accept or reject appointments.

### CODE REVIEW

File: client/src/components/auth/Auth.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #auth {
   padding-top: 5vh;
-  display: flex;
-  flex-direction: column;
+  /* display: flex; */
```
Comment (MackenzieBerliner-Glasser) [inline client/src/components/auth/Auth.css]: commented out code?

Comment (MikeBLambert) [reply client/src/components/auth/Auth.css]: fixed

File: server/src/tests/e2e/requests.test.js [hunk 2]
```diff
@@ -87,4 +87,81 @@ describe('requested appointments routes', () => {
...
+                  requestedNannies: expect.arrayContaining([
+                    expect.objectContaining({ status: 'declined' }),
+                  ]),
+                }),
+              ]),
+            );
+          });
+      });
+    // createdRequestedAppointments[0].requestedNannies[0].status = 'reject';
+    // const rejectedRequest = createdRequestedAppointments[0];
```
Comment (MackenzieBerliner-Glasser) [inline server/src/tests/e2e/requests.test.js]: more commented out code

**Review by MackenzieBerliner-Glasser — COMMENTED** [review]
All file changes look valid, one thing some commented out code.

**Review by MackenzieBerliner-Glasser — APPROVED** [review]

**Pull request merged**